backend/schemas/credit.py

```
@@ -1,31 +1,73 @@
 from pydantic import BaseModel
 from typing import Optional
-import uuid
+from datetime import datetime
+from pydantic import BaseModel, validator
```

Ruff (0.12.2)

Redefinition of unused `BaseModel` from line 1

Remove definition: `BaseModel`

(F811)